source/_components/lightwave.markdown

```diff
@@ -44,10 +44,16 @@ lightwave:
       name: Phone socket
     R2D4:
       name: Torch socket
+    registration:
+      name: Registration
 ```
-
+### `host`
 Where `192.168.1.2` is the ip address of your Lightwave hub.
```

Comment:
Is this IP as example or is this fixed IP?

Reply:
This must be a fixed IP address. I know it would be better if the hub could be discovered but I don't know how to do this (yet).

Reply:
I think it is better to replace it with `IP_ADDRESS`